workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfigurationEditor.cs

```diff
@@ -251,33 +227,48 @@ private void ConfigureBuildOptions(BuildEnvironmentConfig environmentConfigurati
         {
             using (IndentLevelScope(1))
             {
-                var buildOptionsString = EnumFlagToString(environmentConfiguration.BuildOptions);
-
                 EditorGUI.BeginChangeCheck();
-
-                var showBuildOptions = EditorGUILayout.Foldout(environmentConfiguration.ShowBuildOptions,
-                    "Build Options: " + buildOptionsString);
-
+                var showBuildOptions = EditorGUILayout.Foldout(environmentConfiguration.ShowBuildOptions, "Build Options");
                 var newBuildOptions = environmentConfiguration.BuildOptions;
-
                 if (showBuildOptions)
                 {
-                    newBuildOptions = EnumFlagsToggleField(environmentConfiguration.BuildOptions);
+                    using (IndentLevelScope(1))
+                    {
+                        var indentedHelpBox =
+                            new GUIStyle(EditorStyles.helpBox) { margin = { left = EditorGUI.indentLevel * 16 } };
+
+                        using (new EditorGUILayout.VerticalScope(indentedHelpBox))
+                        using (IndentLevelScope(-EditorGUI.indentLevel))
+                        {
+                            var prevValue = (newBuildOptions & BuildOptions.Development) != 0;
+                            var newValue = EditorGUILayout.ToggleLeft("Development Build", prevValue);
+                            if (prevValue != newValue)
+                            {
+                                newBuildOptions ^= BuildOptions.Development;
+                            }
+
+                            prevValue = (newBuildOptions & BuildOptions.EnableHeadlessMode) != 0;
+                            newValue = EditorGUILayout.ToggleLeft("Headless Mode", prevValue);
+                            if (prevValue != newValue)
+                            {
+                                newBuildOptions ^= BuildOptions.EnableHeadlessMode;
+                            }
+                        }
+                    }
                 }
 
+
                 if ((newBuildOptions & BuildOptions.EnableHeadlessMode) != 0 &&
                     (newBuildOptions & BuildOptions.Development) != 0)
                 {
                     EditorGUILayout.HelpBox(
-                        "\nYou cannot have both EnableHeadlessMode and Development in BuildOptions.\n\n" +
-                        "This will crash Unity Editor while building.\n",
+                        "You cannot have EnableHeadlessMode and Development build enabled.\n" +
```

Comment:
The first sentence lost a bit of meaning here; if we're going to remove "both" we may need to find a better alternative instead.

Reply:
guess, we can use the both again ;)